docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -j 16
```

Comment:
16 cores to build a doc? Overkill much? 😱 

p.s. I know, I know, you said don't review...

Reply:
Have you **tried** building the astropy docs? :rofl:

Reply:
Unfortunately, yes. Even on Windows at some point!